### PR TITLE
Fix server telemetry configuration section

### LINKED ISSUE #3547: Update server telemetry config setting in docs
## Description
Starting `prefect server start` I have noticed "sending telemetry information..." message and tried to disable this.

The message looks like:
```shell
apollo_1    | Server ready at http://0.0.0.0:4200 🚀 (version: master)
apollo_1    | Sending telemetry to Prefect Technologies, Inc.: {"source":"prefect_server","type":"startup","payload":{"id":"d3037eea-7232-413f-b7bb-11fb0924993e","prefect_server_version":"master","api_version":"
0.2.0"}}
```

According to https://docs.prefect.io/orchestration/server/telemetry.html I created file `/home/javl/.prefect_server/config.toml` with content:

```toml
[telemetry]
enabled = false
```

and saved the file.

Next time with `prefect server start` I saw the "sending temeletry information..." message again.

The settings seem to be ignore.

## Expected Behavior
When I configure server not to send telemetry, I expect the message not to be shown with next `prefect server start`.

## Reproduction

Some time ago, installed prefect==0.13.1

Played with it few times, stopped.

Later started prefect again, noticed the "Sending telemetry..." message. Stopped the server.

Created the configuration file as instructed in .https://docs.prefect.io/orchestration/server/telemetry.html

Started the prefect server again, saw the "Sending telemetry ..." again, stopped the server.

Updated prefect to the latest version 0.13.11

Started `prefect server start` again, the message "Sending telemetry ..." message still apears there.

## Environment
```shell
$ prefect diagnostics
{
  "config_overrides": {
    "flows": {
      "checkpointing": true
    }
  },
  "env_vars": [],
  "system_information": {
    "platform": "Linux-4.19.0-11-amd64-x86_64-with-debian-10.6",
    "prefect_backend": "server",
    "prefect_version": "0.13.11",
    "python_version": "3.7.4"
  }
}
```

### PR DESCRIPTION
**Pull request by joemirizio**
## Summary

Fixes #3547. Updates the server telemetry config setting in docs.

## Changes

Telemetry configuration in docs.

## Importance

Incorrect setting in docs.

## Checklist

This PR:

- [x] adds new tests (if appropriate)
- [x] adds a change file in the `changes/` directory (if appropriate)
- [x] updates docstrings for any new functions or function arguments, including `docs/outline.toml` for API reference docs (if appropriate)

### CODE REVIEW

**Review by joshmeek — DISMISSED** [review]
Thanks @joemirizio!

File: docs/orchestration/server/telemetry.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ To opt-out of telemetry, add the following to a prefect server configuration fil
 # ~/.prefect_server/config.toml
```
Comment (joshmeek) [inline docs/orchestration/server/telemetry.md:8]:
```suggestion
# ~/.prefect/config.toml
```

Comment (joshmeek) [reply docs/orchestration/server/telemetry.md:8]: One other change on that page 🙂

**Review by joshmeek — APPROVED** [review]

**Comment by joemirizio** [comment]
@joshmeek do you mind tagging this PR with `hacktoberfest-accepted`? It transitioned to "Ineligible Repository" on my dashboard after the `hacktoberfest` topic was removed from the repository. 

It looks like the tags and labels need to stay active for the 14 day waiting period in order to count 😞

**Pull request merged**